demos/rerank/README.md

```diff
@@ -16,142 +16,85 @@ It will create an image called `openvino/model_server:latest`.
 > **Note** Python 3.9 or higher is needed for that step
 Here, the original Pytorch LLM model and the tokenizer will be converted to IR format and optionally quantized.
 That ensures faster initialization time, better performance and lower memory consumption.
-LLM engine parameters will be defined inside the `graph.pbtxt` file.
 
 Install python dependencies for the conversion script:
 ```bash
-export PIP_EXTRA_INDEX_URL="https://download.pytorch.org/whl/cpu"
-pip3 install optimum-intel@git+https://github.com/huggingface/optimum-intel.git  openvino-tokenizers[transformers]==2024.4.* openvino==2024.4.* nncf>=2.11.0 sentence_transformers==3.1.1 openai "transformers<4.45"
+pip3 install -r https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/export-models-script/demos/common/export_models/requirements.txt
 ```
 
 Run optimum-cli to download and quantize the model:
 ```bash
-cd demos/rerank
-convert_tokenizer -o models/BAAI/bge-reranker-large-tokenizer/1 BAAI/bge-reranker-large
-optimum-cli export openvino --disable-convert-tokenizer --model BAAI/bge-reranker-large --task text-classification --trust-remote-code models/BAAI/bge-reranker-large-rerank/1
+wget https://raw.githubusercontent.com/openvinotoolkit/model_server/refs/heads/export-models-script/demos/common/export_models/export_model.py
```

Comment:
Branch will be deleted

Reply:
@kblaszczak-intel what should we specify here to get sphinx documentation to work?

Main branch? - if so, when we create release branch we will refer to incompatible script, way newer than the one release documentation refers to

Reply:
@dkalinowski on nightly build we would use main branch, on 2024 version it will be releases/2024/5

Reply:
What if we split this script to multiple files in the future? Downloading this single script will cause errors. The script exists within the export_models directory with all the neccessary files like requirements.txt.
I think we should add instruction how to enter this directory and run installation scripts within this directory

Reply:
I can confirm what Dariusz said. If you need help in turning all links to 24.5 when the branch is ready, let me know. We can help. I am not sure what other implications derive from it but I'm pretty sure we won't be able to change how links are handled in docs any time soon. Unfortunately.